tests/ui/requirements/flake8.txt

```diff
@@ -1,3 +1,4 @@
 flake8==3.7.7
 flake8-isort==2.7.0
 flake8-docstrings==1.3.0
+pydocstyle<4.0.0
```

Comment:
Worth adding a comment for later maybe?

Reply:
Thanks for the suggestion, I added a comment. While you're there, could you r? this patch? Thanks!